assignments/array-methods.js

```diff
@@ -56,21 +56,25 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach((runner) => { fullName.push(runner.first_name + " " + runner.last_name) });
```

Comment:
```suggestion
runners.forEach((runner) =>  fullName.push(runner.first_name + " " + runner.last_name));
```
{ } are not needed in a single line arrow syntax